modules/master-node-sizing.adoc

```diff
@@ -6,21 +6,36 @@
 [id="master-node-sizing_{context}"]
 =  Control plane node sizing
 
-The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing.
-
-[options="header",cols="3*"]
+The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing. The control plane tests create the following objects across the cluster in each of the namespaces depending on the node counts:
+
+----
+12 imagestreams
+3 buildconfigs
+6 builds
+1 deployment with 2 pod replicas (sleep) mounting two secrets each. deployment-2pod
+2 deployments with 1 pod replicas (sleep) mounting two secrets. deployment-1pod
+3 services, one pointing to deployment-2pod, and other two pointing to deployment-1pod
+3 routes. 1 pointing to the service deployment-2pod and other two pointing to deployment-1pod
+10 secrets. 2 of them mounted by the previous deployments.
```

Comment:
* 12 image streams
* 3 build configurations
* 6 builds

Reply:
* 3 services, 1 pointing to the service `deployment-2pod`, and other 2 pointing to `deployment-1pod`.
* 3 routes, 1 pointing to the service `deployment-2pod` and other 2 pointing to `deployment-1pod`.
* 10 secrets, 2 of which are mounted by the previous deployments.

Reply:
Fixed it, thanks.